service/src/main/java/bio/terra/user/service/user/ProfileService.java

```diff
@@ -29,6 +31,11 @@ public void setProperty(String userId, List<String> path, Object value) {
       throw new InvalidPropertyException("Cannot overwrite the root object.");
     }
 
+    if (path.size() > MAX_PATH_LENGTH) {
+      throw new BadPathException(
```

Comment:
Might be useful to print path

"Path " + path + " violates max path length of: " + MAX_PATH_LENGTH);

Reply:
we might want to truncate the path if we do want to print it so we are not printing tens of thousands of chars.

Reply:
Agree with the truncated path